### PR TITLE
Basic home page modification, button color adjustment

### PR DESCRIPTION
**Pull request by wongloklam456**
### New Layout
Screenshot 2020-04-06 at 18 14 53

### Button Color Fix (hovered and clicked state)
Screenshot 2020-04-06 at 18 07 48

### CODE REVIEW

File: client/src/screens/home.js [hunk 1]
```diff
@@ -1,11 +1,137 @@
...
+  },
+  heroContent: {
+    //backgroundColor: theme.palette.background.paper,
+    padding: theme.spacing(8, 0, 6),
+  },
+  heroButtons: {
+    marginTop: theme.spacing(4),
+  },
+  cardGrid: {
+    backgroundColor: theme.palette.background.paper,
```
Comment (percywtc) [inline client/src/screens/home.js:38]: I would expect the grid container background color to be gray as well?

File: client/src/screens/home.js [hunk 2]
```diff
@@ -1,11 +1,137 @@
...
+    </Typography>
+  );
 }
+
+const useStyles = makeStyles((theme) => ({
+  icon: {
+    marginRight: theme.spacing(2),
+  },
+  heroContent: {
+    //backgroundColor: theme.palette.background.paper,
```
Comment (percywtc) [inline client/src/screens/home.js:31]: Maybe just delete this line lol

File: client/src/screens/home.js (hunk 1 above)
Comment (wongloklam456) [reply client/src/screens/home.js:38]: so you prefer the whole page to be grey?

Comment (percywtc) [reply client/src/screens/home.js:38]: I would say, it seems weird to have a gray gap between the white container frame and the left/right edge of the screen. Perhaps if changing the implementation and not using container later on, we could consider some other colors!

Comment (wongloklam456) [reply client/src/screens/home.js]: done

**Pull request merged**